src-back/app/classes.py

```diff
@@ -301,7 +339,54 @@ def get_user_info(username: str):
 
     @staticmethod
     def route_click_placeholder(username):
-        return {"status": "unknown", "message": "TODO"}
+        cards = [
+            elem[0] for elem in Account.get_cards(username)
+        ]  # turning a list of tuples into a list of integers
+
+        broccoli_amount = 1
+
+        card_id_list = cards
+
+        # critical_click_chances = Account.get_critical_click_chances(username)
+        critical_click_chances = 1
+
+        """Checking for critical click occurence before applying score modifying cards"""
+        for card_id in card_id_list:
+            match card_id:
+                case 13:
```

Comment:
If we had the time, I would have considered enums or a more future proof ID system such as how Minecraft handle it nowadays. It would also make it more readable.